apigee_edge.go

```diff
@@ -22,7 +22,7 @@ import (
 )
 
 const (
-	libraryVersion = "0.1.0"
+	libraryVersion = "0.2.0"
```

Comment:
I'm going to hold off on merging this until I can test this in my terraform apigee provider.  I'm going to do that shortly.